activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/deployer/BpmnDeployer.java

```diff
@@ -507,4 +519,12 @@ public CachingAndArtifactsManager getCachingAndArtifcatsManager() {
     public void setCachingAndArtifactsManager(CachingAndArtifactsManager manager) {
         this.cachingAndArtifactsManager = manager;
     }
+
+    public ProcessDefinitionDiagramHelper getProcessDefinitionDiagramHelper() {
```

Comment:
@daisuke-yoshimoto if we remove all this from here, the engine will not need to depend on the image-generator project. And we can consume the image-generator project as a service.